lib/src/common/util/auth.dart

```diff
@@ -9,26 +11,65 @@ class Authentication {
   /// GitHub Password
   final String? password;
 
+  final String? bearerToken;
+
+  // TODO: mark the pram as `String` to REQUIRE a non-null value.
+  // NEXT major version
   /// Creates an [Authentication] instance that uses the specified OAuth2 [token].
   const Authentication.withToken(this.token)
       : username = null,
-        password = null;
+        password = null,
+        bearerToken = null;
+
+  /// Creates an [Authentication] instance that uses the specified
+  /// [bearerToken].
```

Comment:
Is the line wrap necessary?

Reply:
To keep within 80 columns and keep the `.` – yes!